docs/changelog.rst

```diff
@@ -4,6 +4,26 @@
 Changes in nbformat
 =========================
 
+5.0
+===
+
+`5.0 on GitHub https://github.com/jupyter/nbformat/milestone/5`__
+
+- ``nbformat`` is now Python 3 only (>= 3.5)
```

Comment:
This will sound redundant, but since I know people will read this line out of context, maybe it should read:

```
- Starting with 5.0, ``nbformat`` is now Python 3 only (>= 3.5) 
```

Reply:
I can make that change